### DISCUSSION TITLE
Universal Gcode Sender 2.1.4 NO me abre en win 10

### DISCUSSION THREAD

**Discussion by smarcano2005** [winder/Universal-G-Code-Sender#2462] [Help]
Hola tuve el siguiente problema con Universal Gcode Sender 2.1.4 NO me abre en win 10 descargué la aplicación al principio me abrio pero despues me salio una ventana donde decia que algunos modulos van a estar desabilitado, yo le di clic en aceptar esa ventana se cerro y despues de eso la aplicación, abre la ventana donde se esta cargando y cuando termina de cargar, ya no me abre la aplicacion que solución me sugiere sobre eso, gracias

**Comment by breiler** [comment]
Your settings have probably gotten corrupt, try and delete this folder and then start the application:
`C:/Users/[your username>]/AppData/Roaming/ugsplatform`

**Reply by smarcano2005** [reply]
waoo viendo en el foro otros mensajes parecido a esta falla y precisamente hice eso borre la carpeta de esa direccion C:/Users/[your username>]/AppData/Roaming/ugsplatform y me resolvio la falla gracias por su ayuda

**Reply by smarcano2005** [reply]
una pregunta amigo yo estoy realizando una maquina cnc pero estoy usando arduino UNO y una tarjeta  con modulo L293 , pero como hago para que el programa reconosca mi arduino

**Reply by breiler** [reply]
I am assuming that you are using GRBL on your Arduino. Which version did you install and what does UGS display in the console when you try to connect.

To my knowledge the L293 is not suitable driver for stepper motors. You should try and find a step/dir stepper motor driver.

**Reply by smarcano2005** [reply]
yo estoy usando estas 2 tarjeta 

![arduino uno](https://github.com/winder/Universal-G-Code-Sender/assets/54566357/ffad70a2-a8ea-4af1-b7b8-affa6eb63e7b)
![shiel L293d](https://github.com/winder/Universal-G-Code-Sender/assets/54566357/2ed606d0-3295-4cd1-8c8c-bfd8afbc4ba1)

**Reply by smarcano2005** [reply]
pero la UGS no me la detecta no se como configurar el arduino
